backend/profiles/models.py

```diff
@@ -0,0 +1,3 @@
+from django.db import models
+
+# Create your models here.
```

Comment:
Weren't we going to make a UserProfile model here?

Reply:
My original PR uses the built-in Django auth_users model.  I believe we can add custom fields to it if we want, but I don't think we need to redefine the built-in stuff ourselves.